common/rabbitmq/templates/service.yaml

```diff
@@ -21,6 +21,10 @@ metadata:
     linkerd.io/inject: enabled
 {{- end }}
 spec:
+  type: {{ default "ClusterIP" .Values.service.type }}
```

Comment:
I get this error when generating template with test values
```
Error: template: rabbitmq/templates/service.yaml:24:38: executing "rabbitmq/templates/service.yaml" at <.Values.service.type>: nil pointer evaluating interface {}.type
helm.go:84: [debug] template: rabbitmq/templates/service.yaml:24:38: executing "rabbitmq/templates/service.yaml" at <.Values.service.type>: nil pointer evaluating interface {}.type
```
The same error in ci/concourse/test-helm-chart

Reply:
I'd move the default straight to the values. I.e.
Here simply: `{{ .Values.service.type }}`
And in the values
```
service:
  type: "ClusterIP"
```
That fixes the problem and personally, I'd prefer my defaults to be in the values instead being spread over the templates.

Reply:
hope the last fix addresses all issues